src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -193,19 +283,27 @@ public String getFunctionName() {
         @Override
         public Step newInstance(Map<String,Object> arguments) {
             boolean failFast = false;
+            boolean propagateWorst = false;
             Map<String,Closure<?>> closures = new LinkedHashMap<String, Closure<?>>();
             for (Entry<String,Object> e : arguments.entrySet()) {
                 if ((e.getValue() instanceof Closure)) {
                     closures.put(e.getKey(), (Closure<?>)e.getValue());
                 }
                 else if (FAIL_FAST_FLAG.equals(e.getKey()) && e.getValue() instanceof Boolean) {
                     failFast = (Boolean)e.getValue();
+                } else if (PROPAGATE_WORST_FLAG.equals(e.getKey())
+                        && e.getValue() instanceof Boolean) {
+                    propagateWorst = (Boolean) e.getValue();
```

Comment:
Does it make sense to specify _both_ `failFast: true` and `propagateWorst: true`? If not, this should throw an IAE if you try.

Reply:
> Does it make sense to specify _both_ `failFast: true` and `propagateWorst: true`?

Great point! I don't think this makes sense.

> If not, this should throw an IAE if you try

Done.